pkgs/applications/video/rtabmap/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, opencv, pcl, libusb1, eigen
+, wrapQtAppsHook, qtbase, g2o, ceres-solver, libpointmatcher, octomap, freenect
+, libdc1394, librealsense, libGL, libGLU, vtkWithQt5, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "rtabmap";
+  version = "unstable-2022-02-07";
+
+  src = fetchFromGitHub {
+    owner = "introlab";
+    repo = "rtabmap";
+    rev = "f584f42ea423c44138aa0668b5c8eb18f2978fe2";
+    sha256 = "sha256-xotOcaz5XrmzwEKuVEQZoeq6fEVbACK7PSUW9kULH40=";
+  };
+
+  patches = [
+    # Our Qt5 seems to be missing PrintSupport.. I think?
```

Comment:
Doesn't seem to be the case, `qt5.qtbase.dev` does contain `include/QtPrintSupport` and `qt5.qtbase.out` contains `lib/libQt5PrintSupport.so`.

Reply:
Well, I wrote that patch and comment after trying to get it to work for a decent amount of time so I don't really have a better solution to offer.. I guess the comment could go but I've already started the big rebuild for this package so that will have to wait for later probably.

Reply:
Well, I wrote that patch and comment after trying to get it to work for a decent amount of time so I don't really have a better solution to offer.. I guess the comment could go but I've already started the big rebuild for this package so that will have to wait for later probably.

Reply:
(Sorry about all the duplicates in case you're seeing them, Firefox went crazy under low-memory conditions)